stories/Alerts.story.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styled from "styled-components";
+import { storiesOf } from "@storybook/react";
+import { Alert, AlertSmall } from "../src/index";
+
+const LayoutStyles = styled.div`
+  width: 80%;
+  margin: 0 auto;
+  padding: 100px;
+`;
+
+storiesOf("Alerts", module)
```

Comment:
I would actually make two stories out of this. usually every story demonstrates or tries to demonstrate one  different property. or one different use case if you prefer. 
I know buttons are not like this but its the only one and its legacy. :| Sorry i should have told you that.

Reply:
👍 great point! I've split it into two top level stories and made the other changes if you want to take another look. 

Thanks for the review!!

Reply:
I will review the rest when you commit the changes 👍

Reply:
Ooops haha I pushed to my fork instead of upstream

Reply:
lel